xclim/utils.py

```diff
@@ -64,29 +64,30 @@
              '360_day': 360}
 
 
-def create_ensemble(ncfiles):
+def create_ensemble(ncfiles, mf_flag=False):
     """Create an xarray datset of ensemble of climate simulation from a list of netcdf files. Input data is
     concatenated along a newly created data dimension ('realization')
 
     Returns a xarray dataset object containing input data from the list of netcdf files concatenated along
     a new dimension (name:'realization'). In the case where input files have unequal time dimensions output
     ensemble dataset is created for overlapping time-steps common to all input files
 
-    Notes
-    -----
-    Input netcdf files require equal spatial dimension size (e.g. lon, lat dimensions)
-    If input data contains multiple cftime calendar types they must be at monthly or coarser frequency
-
-
     Parameters
     ----------
+    ncfiles : sequence
+      List of netcdf file paths. If mf_flag is true ncfiles should be a list of lists where
+    each sublist contains input .nc files of a multifile dataset
 
-    ncfiles : list of netcdf file paths (list)
-
+    mf_flag : Boolean . If true climate simulations are treated as multifile datasets before concatenation
```

Comment:
Do you need mf_flag ? If the user sends a list of list, you could apply the mf_flag behavior.